### PR TITLE
Setup leaderboard proyect

### PR DESCRIPTION
**Pull request by Eldonratzlaff**
Setup leaderboard proyect

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
+{
+  "name": "leaderboard",
+  "version": "1.0.0",
+  "description": "",
+  "private": true,
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "webpack"
+  },
```
Comment (MaganaAsiati) [inline package.json]: - You did a great job by using webpack in your project but make sure  that 
1. All webpack-related packages should be included in `package.json` like `webpack`, `webpack-cli`, `webpack-dev-server` etc. Also, those packages should be added as devDependencies
1. Has `npm run ?` scripts like build, start in `package.json`. See the example below.

```json
  "scripts": {
    "dev": "webpack --mode development",
    "build": "webpack --mode production",
    "start": "webpack --config webpack.config.js --watch"
  }
```

Please make sure after implementing the point I listed above for the configuration of webpack. you must be able to open the project with using the command npm run start in your terminal. :+1:

**Review by MaganaAsiati — CHANGES_REQUESTED** [review]
# STATUS: Required Changes ♻️♻️♻️ 
## Hi @Eldonratzlaff , I'm @MaganaAsiati, your reviewer for this PR  :wave: 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

so-close-gif-4

## Highlights

- Descriptive README file  :heavy_check_mark:👨🏻‍💻
- Great work using the right GitHub Flow.:heavy_check_mark:
- No Linter errors :heavy_check_mark:

## Required Changes ♻️

- [ ] You did the great Job by adding readme file but please make sure you use the one provide by microverse and update it with your information :+1: 
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._
- [ ] [OPTIONAL] Please make sure to make your commit messages more descriptive in the future. :heavy_check_mark:👨🏻‍💻
- [ ] kindly  Add a meaningful description to your pull request in the future which gives more information about what you implement in this milestone :heavy_check_mark:👨🏻‍💻


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: README.md [hunk 2]
```diff
@@ -1 +1,164 @@
...
+
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo of my page"></a>
+
+
+> Add a link to your deployed project.
+
+- [Live Demo Link](https://eldonratzlaff.github.io/leaderboard/)
```
Comment (Zuhaib042) [inline README.md:72]: - [OPTIONAL] This link is broken so I will highly recommend you to consider fixing it.

File: README.md [hunk 3]
```diff
@@ -1 +1,164 @@
...
+Original design idea by Cindy Shin in Behance.
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](MIT.md) licensed.
```
Comment (Zuhaib042) [inline README.md:160]: - [OPTIONAL] This link is also broken so I will highly recommend you to fix this and have an MIT license in your repo  to make your repo more professional.

**Review by Zuhaib042 — APPROVED** [review]
Hi @Eldonratzlaff , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
approve
## Highlights
Descriptive README file ✔️👨🏻‍💻
Great work using the right GitFlow.✔️
No Linter errors ✔️


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**